webpack/components/extensions/HostDetails/DetailsTabCards/RegistrationCard.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  List,
+  ListItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import IsoDate from 'foremanReact/components/common/dates/IsoDate';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+
+export const RegisteredBy = ({ user, activationKey }) => {
+  if (user) {
+    return (
+      <DescriptionListDescription>{user}</DescriptionListDescription>
+    );
+  }
+  return (
+    <>
+      <List isPlain>
+        {activationKey.map(key => (
+          <ListItem key={key.id}>
+            <Text component={TextVariants.h4}>Activation key</Text>
```

Comment:
![multi_ak](https://user-images.githubusercontent.com/22042343/166503133-d61c6f9e-0fa2-44e8-a272-c4b4d6f6bfb2.png)

Repeating the 'Activation key' term for multiple keys sorta makes the card too tall if there are more than 2 keys. Probably better to move the `<Text>` outside of the map..

Reply:
extra credit: use `<TranslatedPlural>` and pluralize 'Activation key'